src/modeler/AutoRest.Swagger.Tests/Swagger/Validation/default-value-not-in-enum.json

```diff
@@ -0,0 +1,50 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default value for an ObjectSchema does not appear in enum constraint",
+    "description": "Some documentation.",
+    "version": "2014-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/foo": {
+      "post": {
+        "operationId": "PostFoo",
+        "summary": "Foo path",
+        "description": "Foo operation",
+        "responses": {
+          "default": {
+            "description": "Unexpected error"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Test1": {
+      "description": "Property for foo path 1"
+    },
+    "Test2": {
+      "description": "Property for foo path 2"
+    },
+    "Test": {
+      "type": "string",
+      "description": "Property for foo path",
+      "enum": [
+        "Foo",
+        "Bar"
+      ],
+      "default": "Baz" // not in enum
```

Comment:
JSON doesn't support comments.  :crying_cat_face: 

Some json processors will completely fail to parse with comments.